public/docs/ts/latest/guide/user-input.jade

```diff
@@ -37,27 +40,27 @@ include ../_util-fns
 .l-main-section
 :marked
   ## Get user input from the $event object
-  We can bind to all kinds of events. Let's bind to the keyup event of an input box and replay
+  You can bind to all kinds of events. In this case, you'll bind to the keyup event of an input box and replay
   what the user types back onto the screen.
 
-  This time we'll (1) listen to an event and (2) grab the user's input.
+  This time, you'll listen to an event and grab the user's input.
 +makeExample('user-input/ts/app/keyup.components.ts', 'key-up-component-1-template', 'app/keyup.components.ts (template v.1)')(format=".")
 :marked
   Angular makes an event object available in the **`$event`** variable,
-  which we pass to the component's `onKey()` method.
-  The user data we want is in that variable somewhere.
+  which it passes to the component's `onKey()` method.
```

Comment:
Angular doesn't pass it, our code does. Maybe:

which the code passes as a parameter to ...